src/Simulation/Native/src/simulator/wavefunction.hpp

```diff
@@ -549,6 +550,40 @@ class Wavefunction
         return kernels::jointprobability(wfn_, bs, get_qubit_positions(qs));
     }
 
+    /// \pre: Qubits, listed in `q`, must be unentangled and in state |0>.
+    /// place these `n` qubits into superposition of 2^n basis vectors with (re, im) amplitudes, where the order of
+    /// qubits in array `q` defines (big or little endian?) order of the basis vectors.
```

Comment:
Would it be good to note in this API comment the temporary limitation that subsystem state injection is not yet supported? I could see either way, since that limitation is going away pretty soon.